inc/class-wpseo-options.php

```diff
@@ -2791,192 +2791,195 @@ protected function clean_option( $option_value, $current_version = null, $all_ol
 /*******************************************************************
  * Option: wpseo_ms
  *******************************************************************/
-if ( is_multisite() ) {
```

Comment:
@Rarst I'd have to look in detail, but I suspect that will still make some things go wrong as in the `WPSEO_Options` class, the available instances are used for certain actions and that logic relies on the _ms option not being instantiated.